app/models/patient_business_identifier.rb

```diff
@@ -12,4 +12,13 @@ class PatientBusinessIdentifier < ApplicationRecord
 
   validates :device_created_at, presence: true
   validates :device_updated_at, presence: true
+
+  def shortcode
+    case identifier_type
+    when "simple_bp_passport"
```

Comment:
I think this can be written more simply as `when simple_bp_passport?`

Reply:
I don't follow... is `simple_bp_passport` defined somewhere?

Reply:
Rails enums should also give you booleans for asserting whether the `identifier_type` is of a particular kind like `simple_bp_passport?` automatically?

Reply:
Rails enums should also give you booleans for asserting whether the `identifier_type` is of a particular kind like `simple_bp_passport?` automatically.